yarn-project/sequencer-client/src/block_builder/solo_block_builder.ts

```diff
@@ -128,6 +129,18 @@ export class SoloBlockBuilder implements BlockBuilder {
     // Check txs are good for processing
     this.validateTxs(txs);
 
+    const emptyTxHash = await Tx.emptyTxHash();
+
+    for (const tx of txs) {
+      if (tx.hash.equals(emptyTxHash)) {
```

Comment:
Should we just write a `tx.isEmpty()` function, rather than relying on the empty hash?